playwright-e2e/dsm/component/filters/sections/search/search.ts

```diff
@@ -33,6 +33,8 @@ export class Search {
   public async dates(columnName: string, { from: fromValue, to: toValue, additionalFilters }: Partial<DateConfig>): Promise<void> {
     await this.setAdditionalFilters(columnName, additionalFilters);
 
+    if (!fromValue && !toValue) { return; }
```

Comment:
This check should be moved up to be the first line of code, right?

Reply:
Thanks. 
No, because additional filters can still be applied.